editor/src/core/es-modules/package-manager/built-in-dependencies-list.ts

```diff
@@ -7,8 +7,8 @@ import * as ReactDOM from 'react-dom'
 import * as EmotionReact from '@emotion/react'
 import * as EmotionStyled from '@emotion/styled'
 
-import * as editorPackageJSON from '../../../../package.json'
-import * as utopiaAPIPackageJSON from '../../../../../utopia-api/package.json'
+import editorPackageJSON from '../../../../package.json'
+import utopiaAPIPackageJSON from '../../../../../utopia-api/package.json'
```

Comment:
Was this related?

Reply:
yep, changing the module resolution in tsconfig made webpack realize that this should be a default import